src/sage/categories/lie_algebras.py

```diff
@@ -769,27 +775,27 @@ def _test_distributivity(self, **options):
 
             TESTS::
 
-                sage: L = LieAlgebras(QQ).example()
-                sage: L._test_distributivity()
+                sage: L = LieAlgebras(QQ).example()                                                                     # optional - sage.combinat sage.modules
+                sage: L._test_distributivity()                                                                          # optional - sage.combinat sage.modules
 
```

Comment:
Isn't this too much far to the right?

Reply:
Hmm. I see. This is aligned with other rows below.

Reply:
And it's in a TESTS block of a private function, so it won't show in the HTML